src/module/actor/entity.js

```diff
@@ -282,9 +282,19 @@ export default class OseActor extends Actor {
     const actorData = this.system;
 
     const label = game.i18n.localize(`OSE.roll.hd`);
-    const rollParts = [actorData.hp.hd];
+
+    const rollParts = [];
+
     if (actorType === "character") {
-      rollParts.push(actorData.scores.con.mod * actorData.details.level);
+      const hd = `${actorData.details.level}${actorData.hp.hd.slice(
+        actorData.hp.hd.indexOf("d")
+      )}`;
+      rollParts.push(hd);
+      rollParts.push(
+        `${actorData.scores.con.mod} * ${actorData.details.level}`
```

Comment:
Please see the discussion here going all the way to the end

https://discord.com/channels/929420928528568390/929511321488203819/1061817746238738482

Particularly here, where the Moldvay Basic wording of the rules is written, and we're given to understand that the OSE rules are meant to emulate the Moldvay Basic rules procedure without repeating them word-for-word

![image](https://user-images.githubusercontent.com/1410433/229648221-7852c7d2-7e7d-4a51-b154-350d6d37c8c6.png)

There was a later conversation that I can't find, where I *finally* understood the rules

In short: our current implementation is actually incorrect, due to me choosing a different reading of the rules than was apparently intended. We should not be multiplying the con bonus by level